src/components/Dropdown/Dropdown.vue

```diff
@@ -400,6 +400,7 @@ export default {
         } else if (key === Keys.PageDown) {
           return MenuActions.PageDown;
         } else if (key === Keys.Escape) {
+          $event.stopImmediatePropagation();
```

Comment:
I _think_ probably we should be doing this at the very top of the function.  I can't think off the top of my head why wouldn't wouldn't want to stop propagation for all keys.

Reply:
yeah that is good point - hard to think of examples but I agree